mirdata/core.py

```diff
@@ -76,12 +84,13 @@ class Dataset(object):
 
     Attributes:
         data_home (str): path where mirdata will look for the dataset
+        index (dict or None): the dataset's file index
         name (str): the identifier of the dataset
-        bibtex (str): dataset citation/s in bibtex format
-        readme (str): a link to information about the dataset
-        remotes (dict): data to be downloaded
-        download_info (str): download instructions or caveats
-        track (core.Track): an uninstantiated Track object
+        track_object (mirdata.core.Track or None): an uninstantiated Track object
```

Comment:
track_object is an argument in init (documented there) but not an attribute of the object (which is hidden `_track_object`)

Reply:
^^ i pushed a change that fixes the problem